### ISSUE TITLE
How to toggle chat?

### ISSUE THREAD

**Issue by brylie** [thecodingmachine/workadventure#1172]
We are looking to WorkAdventure as a richer alternative to Slack or other chat spaces. However, the chat feature seems to be a bit hidden and may not allow global discussions. How can I open the global chat to encourage participants to use text chat when needed?

**Comment by brylie** [comment]
I.e. it may feel a bit alien for a person to enter into a relatively large environment and try to find people. Most games have global chat for when there are no players in local regions. Likewise, chat interfaces like IRC and Slack make it easy to send messages to groups of people and read messages that have been sent in the past.

**Comment by jonnytest1** [comment]
> I.e. it may feel a bit alien for a person to enter into a relatively large environment and try to find people. Most games have global chat for when there are no players in local regions. Likewise, chat interfaces like IRC and Slack make it easy to send messages to groups of people and read messages that have been sent in the past.

workadventure currently does not have a rich chatting experience like slack - maybe you can embed a chatting solution in a cowebsite for now ?

**Comment by brylie** [comment]
OK, that sounds like a good idea. I'll try to figure out how cowebsites work

**Comment by Lukas1818** [comment]
I did had a similar problem #1169

**Comment by moufmouf** [comment]
Indeed, we currently don't have a global chat, but it is a feature we will probably be working on in the future!
